concepts/vector.md

```diff
@@ -1,7 +1,33 @@
 ---
 grand_parent: Resources
 parent: Concepts
-layout: coming_soon
 title: Vector
 description:
 ---
+
+A **vector** is a unique fixed-sized list of numbers.
+Each vector number is called coordinate.
+Coordinates define the vector's position in a particular space.
+Vectors represent data in a way that a machine learning model can understand and use.
+In machine translation, vectors represent the words and phrases in a text.
+
+Example of a vector:
+
+`cat` = [0.049, 0.121, 0.503, 0.888]
+
+Each coordinate corresponds to a specific aspect of the word or phrase.
+
+Vector information:
```

Comment:
To avoid potential issues when machine translating the site into languages that don't support references such as "following", we try to introduce lists with complete sentences. Here's the [instruction](https://machinetranslate.org/style#lists).